packages/react-reconciler/src/__tests__/ReactHooks-test.internal.js

```diff
@@ -1754,6 +1756,9 @@ describe('ReactHooks', () => {
     );
     expect(root).toMatchRenderedOutput('loading');
     await Promise.resolve();
+    if (enableNewScheduler) {
+      Scheduler.flushAll();
```

Comment:
All of them in this file are conditional. I only made it not conditional if I wanted to avoid converting the test suite to an `internal` one since those don't run against the public bundles.

Reply:
Seems like if the same behavior works in both, we should commit it unconditionally so that we know we're testing the same thing in both and not covering up an unknown discrepancy.